corehq/apps/reports/filters/forms.py

```diff
@@ -45,8 +45,7 @@ def display_lang(self):
         """
             This should return the lang code of the language being used to view this form.
         """
-        # todo make this functional
-        return 'en'
+        return self.request.couch_user.language or 'en'
```

Comment:
do we ever have to worry about `self.request` or `self.request.couch_user` being `None`? might be safe to use `getattr`, or if you're convinced they're always set then feel free to merge as is